.github/workflows/rundoc.yml

```
@@ -12,30 +12,52 @@ jobs:
       contents: write
       pull-requests: write
     steps:
+      - name: Get token for GH application (Linguist)
+        uses: actions/create-github-app-token@v1
+        id: generate-token
+        with:
+          app-id: ${{ vars.LINGUIST_GH_APP_ID }}
+          private-key: ${{ secrets.LINGUIST_GH_PRIVATE_KEY }}
+
       - uses: actions/checkout@v4
         with:
           ref: ${{ github.head_ref }}
+          token: ${{ steps.generate-token.outputs.token }}
+
       - uses: ruby/setup-ruby@v1
         with:
           ruby-version: '3.3.2'
           bundler-cache: true # do run 'bundle install' and cache
           working-directory: .rundoc-workspace
       - uses: buildpacks/github-actions/setup-pack@v5.7.1
-      # [CONFIG] here for the path of what we want to generate
+
+
+      # [CONFIG] different languages set up here
       - run: bundle exec rundoc ../docs/src/ruby/ruby_tutorial.md --on-success-dir ../docs/ruby --on-failure-dir ../docs/fail --force
         working-directory: .rundoc-workspace
         continue-on-error: true
       - run: rm -rf ruby-getting-started
         working-directory: docs/ruby
         continue-on-error: true
-      # [CONFIG] then here move and rename README.md to the appropriate place
-      - run: echo "NOW=$(date +'rundoc-linux/%s')" >> $GITHUB_ENV
-      - run: echo "HUMAN_DATE=$(date +'update tutorials - %Y %b %d')" >> $GITHUB_ENV
+
+
       - name: Create Pull Request
+        id: pr
         uses: peter-evans/create-pull-request@v6
         with:
-          branch: ${{ env.NOW }}
+          token: ${{ steps.generate-token.outputs.token }}
+          branch: rundoc-update
+          delete-branch: true
           base: main
-          commit-message: run rundoc
-          title: ${{ env.HUMAN_DATE }}
+          commit-message: run rundoc on tutorials
```

I think the commit message title should match the PR's title (with `peter-evans/create-pull-request` it requires specifying the value twice; the `title` config option only applies to the PR title) :-)

```suggestion
          commit-message: Update tutorials
```